azure-pipeline-integration-test-steps-template.yaml

```diff
@@ -14,7 +14,7 @@ steps:
   condition: eq(variables['AdditionalIntegrationTestArguments'], '')
   workingDirectory: ./LoRaEngine/test/LoRaWan.IntegrationTest
   failOnStderr: true
-  timeoutInMinutes: 60 # up to 1 hour, increase if we need more
+  timeoutInMinutes: 120 # up to 1 hour, increase if we need more
```

Comment:
Comment says up to 1 hour, should be up to 2 hours

Reply:
Thanks. Fixed.